iscript/src/iscript/autograph.py

```diff
@@ -32,6 +32,10 @@
 # Blessed files call the other widevine files.
 _WIDEVINE_BLESSED_FILENAMES = (
     # plugin-container is the top of the calling stack
```

Comment:
Can you move the comment to stay next to plugin-container?  And/or update it to explain what media plugin helper is?

Reply:
Added comment to explain these are the files that load the Widevine CDM and what the new entries are for.